.github/scripts/label_utils.py

```diff
@@ -83,6 +83,13 @@ def gh_add_labels(
     )
 
 
+def gh_remove_label(org: str, repo: str, pr_num: int, label: str) -> None:
+    gh_fetch_json(
```

Comment:
Why not use `gh_fetch_url` here since you do not care about the output anyway?

Reply:
true I was just copying over pattern